operator/v1/types_ingress.go

```diff
@@ -131,6 +131,15 @@ type IngressControllerSpec struct {
 	// tlsSecurityProfile specifies settings for TLS connections for ingresscontrollers.
 	//
 	// If unset, the default is based on the ingresses.config.openshift.io/cluster resource.
+	// If the default is used, and the ingresses.config.openshift.io/cluster resource
+	// diverges from the ingresscontroller's effective default, the ingresscontroller will
+	// continue to use the defaults applied at the time of creation.
```

Comment:
I think based on the most recent discussion, this is false: the operator will eschew whatever has been persisted in the ingresscontroller's status and reconcile the effective TLS profile.

Reply:
I agree. Removed this entire section.